assets/src/design-system/components/input/useNumericInput.js

```diff
@@ -33,7 +33,7 @@ export const useNumericInput = ({
 }) => {
   const inputRef = useRef(null);
   const oldValue = useRef(value);
-  const revertToOriginal = useRef(false);
+  const revertToOriginal = useRef(true);
```

Comment:
changes in this file allow the input to not submit if the value remains unchanged.

Reply:
Yep, this is a bit confusing, this file is currently changed in different ways for the same reason.

Reply:
I rebased this PR from main and have spent some time testing the 'mixed' inputs and such. I think what we have in this PR is a needed addition. 

@miina @barklund I won't even begin to pretend to be an expert on the end goal here but I'd like to be able to move this PR to ready for QA if possible. Let me know how you want to proceed.

Reply:
In theory, it shouldn't be needed -- will test it out and keep it/remove as needed.
There will be a separate issue to address all the 'mixed' inputs so it's OK to continue without that currently.

Reply:
I've reverted the changes here and added the use of `isIndeterminate` as used elsewhere.
There will be a separate ticket for ensuring the "mixed" value is working as expected across the panels -- if we'd like another approach instead of `isIndeterminate` we can then change it for all the panels together for consistency.